addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2414,22 +2414,19 @@ export class OdooEditor extends EventTarget {
         if (this._isResizingTable || !traversedNodes.some(node => !!closestElement(node, 'td') && !closestElement(node, '[data-oe-protected="true"]'))) {
             return false;
         }
-        let range;
-        if (selection.rangeCount > 1) {
-            // Firefox selection in table works with multiple ranges.
-            const startRange = getDeepRange(this.editable, {range: selection.getRangeAt(0)});
-            const endRange = getDeepRange(this.editable, {range: selection.getRangeAt(selection.rangeCount - 1)});
-            range = this.document.createRange();
-            range.setStart(startRange.startContainer, 0);
-            range.setEnd(endRange.startContainer, 0);
-        } else {
-            // We need the triple click correction only for a bug in firefox
-            // where it gives a selection of a full cell as tr 0 tr 1. The
-            // correction makes it so it gives us the cell and not its neighbor.
-            // In all other cases we don't want to make that correction so as to
-            // avoid flicker when hovering borders.
-            range = getDeepRange(this.editable, { correctTripleClick: anchorNode && anchorNode.nodeName === 'TR' });
+        if (this.isFirefox) {
+            if (selection.rangeCount > 1) {
+                this.historyResetLatestComputedSelection(true);
```

Comment:
How does this work?

Reply:
As I have mentioned before, when we select multiple `tds` in the table with mouse, the selection changes to the `TR`, resulting in the loss of our original selection. Therefore, what I am doing is retrieving the original selection when the selection moves to the TR using `this.historyResetLatestComputedSelection(true) `, so we don't lose it.

Reply:
But if the original selection was saved, that means at one point we had the correct selection. What changed it? Is it the browser or something in our code?

Reply:
Yes, until the second `td` is selected, we have the correct selection. However, when the second `td` is selected, the selection shifts to the `tr`. I believe this behavior is caused by the browser and is not related to our code.

Reply:
This requires a comment in the code because there is no way to guess this particular case just by reading the code.